pyhf/interpolators/code1.py

```diff
@@ -98,9 +99,9 @@ def product(self, down, nom, up, alpha):
         delta_up = up / nom
         delta_down = down / nom
         if alpha > 0:
-            delta = delta_up ** alpha
+            delta = math.pow(delta_up, alpha)
         else:
-            delta = delta_down ** (-alpha)
+            delta = math.pow(delta_down, (-alpha))
```

Comment:
this is not code that's actually used in pyhf proper, but just reference code we check our production code against. I'm fine with changing it, but it'll not affect e.g. fit performance

Reply:
(same is true also for the other _slow_x commands

Reply:
I know. My reasoning above is:
> The slow implementation of the code for validation is also updated (as it is in code 1 as well) to have consistent style across the files.

Reply:
hm, ok need to re-read (sorry) I didn't see anything in the PR that affected the non-slow impls (because I would have thought those all go through `tensorlib` anyways)

Reply:
This commit is the one that changes the non-slow portion of code 4: https://github.com/diana-hep/pyhf/pull/506/commits/8844dbe869c4415a46f9c08c15c451d752790741

Reply:
ah I see, thanks. I think I missed this because it's in the `__init__`, i.e. a one-time computation.

Reply:
Yeah, this is a _very_ minor improvement PR.